src/mad_spark_alt/core/unified_evaluator.py

```
@@ -0,0 +1,274 @@
+        self,
+        response: str
+    ) -> Tuple[Dict[str, float], Dict[str, str]]:
+        """Parse the LLM evaluation response into scores and explanations."""
+        scores = {}
+        explanations = {}
+        
+        criteria = ["novelty", "impact", "cost", "feasibility", "risks"]
+        
+        for criterion in criteria:
+            # Look for pattern like "Novelty: 0.7 - explanation"
+            pattern = f'{criterion}:\\s*([0-9.]+)\\s*[-–]\\s*(.+?)(?={"|".join(criteria)}:|$)'
```

Ruff (0.12.2)

String contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)